src/services/SlideService.ts

```diff
@@ -0,0 +1,4 @@
+import SlideRepository from '../repositories/SlideRepository'
+const deletePresentation = async (id: string) =>
```

Comment:
check if the slides are there, if not throw a generic error stating that they are not found.